nautilus_trader/adapters/bybit/common/enums.py

```diff
@@ -537,50 +554,27 @@ def parse_bybit_order_type(
     def parse_nautilus_time_in_force(self, time_in_force: TimeInForce) -> BybitTimeInForce:
         try:
             return self.nautilus_to_bybit_time_in_force[time_in_force]
-        except KeyError:
+        except KeyError as exec:
```

Comment:
To align with Rust, we standardized on using `as e` for the exception variable name. (I'll refactor this after merging.)